code/game/objects/items/storage/backpack.dm

```diff
@@ -424,6 +424,44 @@
 	name = "surgical duffel bag"
 	desc = "A large duffel bag for holding extra medical supplies - this one seems to be designed for holding surgical tools."
 
+/obj/item/storage/backpack/duffelbag/explorer
+	name = "explorator's duffel bag"
+	desc = "A large duffel bag for holding extra exotic treasures."
+	icon_state = "duffel-explorer"
+	inhand_icon_state = "duffel-explorer"
+
+/obj/item/storage/backpack/duffelbag/hyd
```

Comment:
We don't gain anything from short names like this. I know the other ones have similar, but please extend these out (`hydroponics`, `chemist`, `genetics`) etc.

Reply:
Changes done